tests/serde.rs

```diff
@@ -0,0 +1,16 @@
+#![cfg(feature = "serde")]
+
+use std::assert_eq;
```

Comment:
Sorry for the nitpicking, but isn't this one part of the prelude?

Reply:
Fixed